torch/distributed/device_mesh.py

```diff
@@ -506,7 +506,11 @@ def _init_process_groups(self):
                 default_group = _get_default_group()
                 ranks = list(range(get_world_size()))
                 dim_group = (
-                    new_group(backend="cpu:gloo,cuda:nccl", ranks=ranks)
+                    new_group(
+                        backend="cpu:gloo,cuda:nccl",
+                        ranks=ranks,
+                        group_desc="mesh_dim_world",
```

Comment:
Can we give it a more meaningful name? Like default or mesh_all? what does "dim_world" mean to users?

Reply:
Can we name it "mesh_default" or just "default"?